doc/installation.md

```diff
@@ -98,8 +98,9 @@ Install parallel-bilby:
 
 Install pymultinest 
 
+For using the PyMultinest library, it is required to get and compile the Multinest library as well. Instructions for the same are given here:
 
-* $ conda install -c conda-forge pymultinest
```

Comment:
I think we should leave the conda environment as the first pass, and as a backup, installing from source. It clearly depends on the Linux distro, environment, etc.

Reply:
Made some changes. Please let me know if this is sufficient.